src/linpred.jl

```diff
@@ -157,11 +162,32 @@ function delbeta!(p::DensePredChol{T,<:Cholesky}, r::Vector{T}, wt::Vector{T}) w
 end
 
 function delbeta!(p::DensePredChol{T,<:CholeskyPivoted}, r::Vector{T}, wt::Vector{T}) where T<:BlasReal
-    cf = cholfactors(p.chol)
```

Comment:
@andreasnoack It permutes delbeta rather than doing `ldiv` directly that fixed the slow convergence issue